flashCards.rb

```diff
@@ -0,0 +1,123 @@
+require "pry"
+class Flashcard 
+	attr_accessor :front, :back
+	def initialize
+
+	@front = "front"
+    @back = "back"
+    @new_front = []
+    @new_back = []
+
+    @@default_cards = [@front, @back]
+    @@new_cards = [@new_front, @new_back]
```

Comment:
the value of `@@newcards` is going to be overwritten or reassigned each time a new card is made. I'd move `@@new_cards` out of initialize (above it).

https://stackoverflow.com/a/15773671